api/src/main/resources/cb_search_api.yaml

```diff
@@ -388,12 +400,20 @@ definitions:
       - operator
       - operands
     properties:
-      operator:
+      name:
         type: string
+        description: the name of the attribute
+      operator:
+        $ref: "#/definitions/Operator"
       operands:
         type: "array"
         items:
           type: string
+      conceptId:
+        description: The concept id that maps to concept table.
```

Comment:
So this is a concept ID for a concept related to the concept on the containing SearchParameter? Can you give an example in the documentation?

Reply:
So Blood Pressure has 2 concept ids(systolic and diastolic). Therefore these 2 values can't reside in the concept_id column in the criteria table. So I've put a conceptId property on the Attribute object to account for this issue. It's not optimal, but it was the least impact to get this story complete. Let me know if you have any better ideas?

Reply:
OK... still not quite following why this needs to be on Attribute. Seems like maybe you could just special case the BP criteria on the client and server?